models/netsuite2/stg_netsuite2__accounts.sql

```diff
@@ -29,6 +29,7 @@ final as (
         accttype as account_type_id,
         sspecacct as special_account_type_id,
         fullname as name,
+        accountsearchdisplaynamecopy as display_name,
```

Comment:
If you were to do a query on our internal data, it looks as if `accountsearchdisplayname` is a coalesce of `acctnumber` and `accountsearchdisplaynamecopy`. I presume since `acctnumber` is already present in this model`, the newly converted `accountsearchdisplaynamecopy` to `display_name` field provides the useful information not provided. 

However, `display_name` might be confusing, since technically `accountsearchdisplayname` is supposed to be that. So I wonder if renaming it to `account_name` might be better.

Reply:
The value in `accountsearchdisplaynamecopy` is presented in the UI as `name`. However, in the `stg_netsuite2__accounts` model, `fullname` was already aliased to `name`.

@fivetran-avinash You are correct that `accountsearchdisplayname` is a concatenation of `acctnumber` and `accountsearchdisplaynamecopy`. However that value is not in the Netsuite UI. I don't think aliasing `accountsearchdisplaynamecopy` as `display_name` should cause confusion. Calling it `name` would be preferable but that could cause breakage.

Reply:
@jmongerlyra Thanks for looking into this! Weird that they would have a a field called `accountsearchdisplayname` not available in the UI, but if that is the case then `display_name` makes more sense. Couldn't use `name` because we already have a field named `name` as such.